pkgs/applications/emulators/craftos-pc/test-eval-hello-world/default.nix

```
@@ -0,0 +1,16 @@
+{ stdenv
+, craftos-pc
+}:
+
+stdenv.mkDerivation {
+  name = "craftos-pc-test-eval-hello-world";
+  meta.timeout = 60;
+  buildCommand = ''
+    export HOME=$(pwd)
+    mkdir $HOME/.local $HOME/.config
+    export XDG_CONFIG_DIR=$HOME/.config
+    export XDG_DATA_DIR=$HOME/.local
+    ${craftos-pc}/bin/craftos --headless --script ${./init.lua} | grep "Hello Nixpkgs!" > /dev/null
```

```suggestion
    craftos --headless --script ${./init.lua} | grep "Hello Nixpkgs!" > /dev/null
```
please add to nativeBuildInputs